cmd/crank/version/version.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2023 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package version contains version cmd
+package version
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alecthomas/kong"
+	"github.com/pkg/errors"
+
+	"github.com/crossplane/crossplane/internal/version"
+)
+
+const (
+	errGetCrossplaneVersion = "unable to get crossplane version"
+)
+
+// Cmd represents the version command.
+type Cmd struct {
+	Client bool `env:"" help:"If true, shows client version only (no server required)." json:"client,omitempty"`
+}
+
+// Run runs the version command.
+func (c *Cmd) Run(k *kong.Context) error {
+	fmt.Fprintln(k.Stdout, "Client Version: "+version.New().GetVersionString())
+	if c.Client {
+		return nil
+	}
+
+	ctx := context.Background()
```

Comment:
Could be worth adding a timeout to this context, just in case. I think a fixed value (e.g. 30s) would be fine.

Reply:
@negz any reason why top and trace using a background context without any timeouts ?